.github/workflows/test_released_action.yml

```
@@ -12,7 +12,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4.1.1
         
-      - name: CCS Build action step
-        uses: apollo-fire/gtest-action@v0.0.3
+      - name: GTest Action
+        uses: apollo-fire/gtest-action@v0.0.4
         with:
           test-path: 'test/working_unittest_project'
+
+      - name: GTest Action with existing CMake file
+        uses: apollo-fire/gtest-action@v0.0.4
+        with:
+          test-path: 'test/working_unittest_project_with_existing_cmake'
```

Might be worth also testing a hybrid usage, there could be a scenario where both are in use if Apollo opt to transition to the new version in the future.
```suggestion
      - name: GTest Action with existing CMake file
        uses: apollo-fire/gtest-action@v0.0.4
        with:
          test-path: 'test/working_unittest_project_with_existing_cmake'

      - name: GTest Action with mix of projects with and without existing CMake file
        uses: apollo-fire/gtest-action@v0.0.4
        with:
          test-path: "test/working_unittest_project;\
            test/working_unittest_project_with_existing_cmake"
```

Careful with the quotes in the suggestion, changing the double quotes to singles could impact line wrapping by introducing spaces where they are not intended!

---

Good idea! I'll implement that now.